plugins/modules/remote_management/redfish/ilo_redfish_command.py

```diff
@@ -0,0 +1,341 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2017-2018 Dell EMC Inc.
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: ilo_redfish_command
+short_description: Manages Out-Of-Band controllers using Redfish APIs
+description:
+  - Builds Redfish URIs locally and sends them to remote OOB controllers to
+    perform an action.
+options:
+  category:
+    required: true
+    description:
+      - Category to execute on OOB controller
+    type: str
+  command:
+    required: true
+    description:
+      - List of commands to execute on OOB controller
+    type: list
+    elements: str
+  baseuri:
+    required: true
+    description:
+      - Base URI of OOB controller
+    type: str
+  username:
+    required: false
+    description:
+      - Username for authenticating to iLO.
+    type: str
+  password:
+    required: false
+    description:
+      - Password for authenticating to iLO.
+    type: str
+  auth_token:
+    required: false
+    description:
+      - Security token for authenticating to iLO.
+    type: str
+  timeout:
+    required: false
+    description:
+      - Timeout in seconds for HTTP requests to iLO.
+    default: 60
+    type: int
+  bios_attributes:
+    required: false
+    description:
+      - BIOS attributes that needs to be verified in the given server
+    type: dict
+  raid_details:
+    required: false
+    description:
+      - RAID details that need to be verified in the given server
+    type: list
+    elements: dict
+  service_attributes:
+    required: false
+    description:
+      - service BIOS attributes that needs to be verified in the given server
+    type: dict
+  uefi_boot_order:
+    required: false
+    description:
+      - Input UEFI Boot Order
+    type: list
+    elements: str
+  logical_drives_names:
+    required: false
+    description:
+      - List of names of logical drives to be fetched from the given server
+    type: list
+    elements: str
+  cert_file:
+    required: false
+    description:
+      - absolute path to the server cert file
+    type: str
+  key_file:
+    required: false
+    description:
+      - absolute path to the server key file
+    type: str
+author:
+  - Gayathiri Devi Ramasamy (@Gayathirideviramasamy)
+  - T S Kushal (@TSKushal)
+  - Varni H P (@varini-hp)
+  - Prativa Nayak (@prativa-n)
+'''
```

Comment:
This module (through the module utils) has many requirements. Please document them.

Reply:
Can you give us an example?